src/Common/src/Interop/Windows/kernel32/Interop.COPYFILE2_EXTENDED_PARAMETERS.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+#pragma warning disable CS0649 // never assigned to warning
```

Comment:
Marking the struct as `[StructLayout(LayoutKind.Sequential)]` is better way to disable this warning.